### ISSUE TITLE
如何基于jax的sum函数，写一个两方数据密态求和的算法

### ISSUE THREAD

**Issue by daydayuphere** [secretflow/secretflow#793]
![image](https://github.com/secretflow/secretflow/assets/132658460/a7554d21-52a5-40bf-8ff5-22ed60eee1a4)

**Comment by daydayuphere** [comment]
联合统计的函数，方差，中位数，都存在这个问题

**Comment by anakinxc** [comment]
Hi @daydayuphere 

可以先把 p1 p2 concatenate 然后再 sum

**Comment by daydayuphere** [comment]
def sum(p1, p2):
    return jax.numpy.sum(jax.numpy.concanate((p1,p2), axis=0))
是这样吗

**Comment by anakinxc** [comment]
> def sum(p1, p2):
> return jax.numpy.sum(jax.numpy.concanate((p1,p2), axis=0))
> 是这样吗

嗯，是的

**Comment by daydayuphere** [comment]
这个问题解决了，谢谢老师。
还有一个小问题，add函数是双目运算符，两方相加可以用jax.numpy.add(p1 , p2)，试了可以
那三方的数据相加要如何写呢？

**Comment by anakinxc** [comment]
> 这个问题解决了，谢谢老师。
> 还有一个小问题，add函数是双目运算符，两方相加可以用jax.numpy.add(p1 , p2)，试了可以
> 那三方的数据相加要如何写呢？

p1 和 p2 先相加，结果再和 p3 相加

**Comment by daydayuphere** [comment]
多谢老师
